src/styles/SearchBar.module.css

```diff
@@ -0,0 +1,28 @@
+
+.data-results {
```

Comment:
@istoga For this project, we are using `camelCase` for the CSS properties; I apologize that I insisted (ignorantly) on `kebab-case` earlier.

Using `camelCase` will allow us to reference the CSS properties in our files as such:

```html
<main className={styles.camelCasedProperty}>
```

However, I know that you followed some direction from @jjleigh and @annonymousauthority with this component, so if the `camelCasing` does not work, leave your current implementation in place and we will discuss in Sprint Planning.